README.md

```diff
@@ -49,14 +49,8 @@ will work:
 
     sudo pip install -r requirements.txt
 
-The Python + cmake interaction is handled in a separate project called
-[pycmake](https://github.com/Statoil/pycmake); you can either install
```

Comment:
Should we not still mention `pycmake` in the README?

Reply:
> Should we not still mention pycmake in the README? 

My suggestion is no; the problem is that `libres` and `ert` will use the `cmake` modules carried and installed by `libecl`; i.e. the dependency to these files is stronger than what is communicated through the use of a submodule.

The best thing would in my opinion be if we could deploy it to `pypi` - then we could add `pycmake` in the `requirements.txt` independently for `libecl, libres` and `ert` - but that requires some effort; so my suggestion is the current PR. No strong feelings though.

Reply:
There is no changes to the pycmake code, we could even duplicate it across the three repos. That way we simplify the build somewhat - I would actually vote for that.

Reply:
minor comment: there is a quote charachter `"` after `build system`that is not matched anywhere

As was discussed earlier today, replicating the pycmake code can be a good option for the time being